packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -54,6 +54,10 @@ export interface IDPointType {
   values: { 0: number; 1: number; data: {} };
   xVal: number | Date;
 }
+export interface IMapXToDataSet {
```

Comment:
Can you tell what all scenarios you have tested. We need to ensure that this change does not regress any existing functionality.

Reply:
I have tested all the existing examples. The styled example of AreaChart cover the corner case of algorithm.